src/build/env.py

```diff
@@ -137,78 +144,58 @@ def install(self, requirements):  # type: (Iterable[str]) -> None
-        """
-        import venv
-
-        venv.EnvBuilder(with_pip=False).create(path)
-        executable = _find_executable(path)
-
-        # Scenario 1: pip is available (either installed or via pth file) within the python executable alongside
-        # this projects environment: in this case we should be able to import it
-        if pip is not None:
-            pip_executable = sys.executable
-        else:
-            # Scenario 2: this project is installed into a virtual environment that has no pip, but the system has
-            # Scenario 3: there's a pip executable on PATH
-            # Scenario 4: no pip can be found, we might be able to provision one into the build env via ensurepip
-            cmd = [executable, '-Im', 'ensurepip', '--upgrade', '--default-pip']
-            try:
-                subprocess.check_call(cmd, cwd=path)
-            except subprocess.CalledProcessError:  # pragma: no cover
-                pass  # pragma: no cover
-            # avoid the setuptools from ensurepip to break the isolation
-            subprocess.check_call([executable, '-Im', 'pip', 'uninstall', 'setuptools', '-y'])
-            pip_executable = executable
-        return executable, pip_executable
-
-    def _find_executable(path):  # type: (str) -> str
-        """
-        Detect the executable within a virtual environment.
+def _find_executable_and_scripts(path):  # type: (str) -> Tuple[str, str]
+    """
+    Detect the executable within a virtual environment.
 
-        :param path: the location of the virtual environment
-        :return: the python executable
-        """
-        config_vars = sysconfig.get_config_vars().copy()  # globally cached, copy before altering it
-        config_vars['base'] = path
-        env_scripts = sysconfig.get_path('scripts', vars=config_vars)
-        if not env_scripts:
-            raise RuntimeError("Couldn't get environment scripts path")
-        exe = 'pypy3' if platform.python_implementation() == 'PyPy' else 'python'
-        if os.name == 'nt':
-            exe = '{}.exe'.format(exe)
-        executable = os.path.join(path, env_scripts, exe)
-        if not os.path.exists(executable):
-            raise RuntimeError('Virtual environment creation failed, executable {} missing'.format(executable))
-        return executable
+    :param path: the location of the virtual environment
+    :return: the python executable
+    """
+    config_vars = sysconfig.get_config_vars().copy()  # globally cached, copy before altering it
+    config_vars['base'] = path
+    env_scripts = sysconfig.get_path('scripts', vars=config_vars)
+    if not env_scripts:
+        raise RuntimeError("Couldn't get environment scripts path")
+    exe = 'pypy3' if platform.python_implementation() == 'PyPy' else 'python'
+    if os.name == 'nt':
+        exe = '{}.exe'.format(exe)
+    executable = os.path.join(path, env_scripts, exe)
```

Comment:
`env_scripts` is relative to `path`:

```suggestion
    executable = os.path.join(env_scripts, exe)
```